### PR TITLE
feature/home

### PR DESCRIPTION
**Pull request by pakoval**

### CODE REVIEW

File: src/components/Button.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <button
+    :type="type"
+    class="btn"
+    :class="name ? [`btn--${name}`] : null"
```
Comment (silchencko) [inline src/components/Button.vue]: Try to keep this logic in a computed property. It might look a little neater

Comment (silchencko) [reply src/components/Button.vue]: This is rather our project good practice

File: src/components/PricingCard.vue [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="pricing-card">
+    <p
+      class="pricing-card__title"
+      :class="tariff.isHot ? 'pricing-card__title--hot': null"
```
Comment (silchencko) [inline src/components/PricingCard.vue]: The same computed prop

File: src/components/Statistics.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <section class="statistics">
+    <div
+      v-for="item in statistics"
+      :key="item.num"
```
Comment (silchencko) [inline src/components/Statistics.vue]: Is 'num' unique?

File: src/styles/components/pricing-card.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
+.pricing-card {
```
Comment (silchencko) [inline src/styles/components/pricing-card.scss:1]: As far as I understand you've decided to keep all styles in separate css files?

**Review by silchencko — COMMENTED** [review]
Looks realy good. Please, leave your feedback on my comments and I'll be ready to approve

File: src/styles/components/pricing-card.scss (hunk 4 above)
Comment (pakoval) [reply src/styles/components/pricing-card.scss:1]: Yes, I create separate files for components or pages

File: src/components/Statistics.vue (hunk 3 above)
Comment (pakoval) [reply src/components/Statistics.vue]: In this case is unique, but when enter same num in object therefore will be not unique

File: src/components/Button.vue (hunk 1 above)
Comment (pakoval) [reply src/components/Button.vue]: i brought out logic from template to computed property

**Review by silchencko — APPROVED** [review]
I appreciate bem in classes. The code looks organized

**Pull request merged**